sockshop/payment/service_unit_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 )
 
-func TestAuthorise(t *testing.T) {
+func TestUnitAuthorise(t *testing.T) {
```

Comment:
I agree with @peterbourgon. Common practice in Go is just to name the test after the function/method you're testing.